qa/tasks/cephfs/filesystem.py

```diff
@@ -629,13 +633,22 @@ def create(self):
         else:
             data_pool_name = self.data_pool_name
 
+        # will use the ec pool to store the data and a small amount of
+        # metadata still goes to the primary data pool for all files.
+        if not self.metadata_overlay and self.ec_profile and 'disabled' not in self.ec_profile:
+            self.target_size_ratio = 0.05
+
         log.debug("Creating filesystem '{0}'".format(self.name))
 
         self.mon_manager.raw_cluster_cmd('osd', 'pool', 'create',
-                                         self.metadata_pool_name, self.pgs_per_fs_pool.__str__())
+                                         self.metadata_pool_name,
+                                         '--pg_num_min', self.pg_num_min.__str__())
```

Comment:
instead of calling the magic method directly, i'd suggest put something like
```suggestion
                                         '--pg_num_min', str(self.pg_num_min))
```

Reply:
Done.